vault/tests/conftest.py

```diff
@@ -77,7 +77,7 @@ def __call__(self):
             user = getpass.getuser()
             chown_args = ['chown', user, self.token_file]
 
-            if user != 'root':
+            if user != 'root' and running_on_ci():
```

Comment:
You don't need sudo on mac?

Reply:
It's running fine without sudo on mac. What about windows ?

Reply:
No, and there is no sudo 😄